.eslintrc.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  env: {
+    browser: false,
+    commonjs: true,
+    es6: true,
```

Comment:
```suggestion
    es2020: true,
```

Reply:
To be in sync with the `parserOptions`.
But for consistency with the node version we should use `es2019` until we move to node 14.